### PR TITLE
speech: add a readme for each example

### PR DESCRIPTION
**Pull request by danicat**
I've added a README.md file for each Speech API example. I've also moved the raw audio files to a testdata directory just like the Vision API examples to avoid duplication. Solves issue #235.

### CODE REVIEW

File: speech/speech_quickstart/main.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ func main() {
 	}
 
 	// Sets the name of the audio file to transcribe.
-	filename := "/path/to/audio.raw"
```
Comment (broady) [inline speech/speech_quickstart/main.go]: I think these are fine, actually. For the quickstarts, we do expect people to modify the programs. I'd rather not make them too specific to this repo.

**Comment by broady** [comment]
READMEs look good, but I don't think the deletion/moving of the raw audio files is an improvement. Could you revert that?

File: speech/speech_quickstart/main.go (hunk 1 above)
Comment (danicat) [reply speech/speech_quickstart/main.go]: ok! I'll revert to the original

**Comment by danicat** [comment]
I will, but I wonder what would be the benefit of having several copies of the same file all over the place? The same thing (having a testdata folder) was done for the vision API and I thought it was a good idea.

**Comment by broady** [comment]
Hm, OK. If they're all exactly the same, sounds good to me.

**Comment by danicat** [comment]
Yes, they are all the same. 

And, yes, *now* you can assume the integration tests are ok. 

I didn't know this limitation about Travis CI. I'm still learning how to contribute to this repo, sorry about the trouble!

**Comment by broady** [comment]
Thank you! I can add you as a contributor so you can push to branches in this repo rather than to your fork. Then the integration tests will run.

**Comment by danicat** [comment]
Oh, great! Thank you!

**Pull request merged**